frontend/src/components/common/Address.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import * as Strings from "@/constants/strings";
+import { Icon } from "antd";
+import { formatPostalCode } from "@/utils/helpers";
+
+const propTypes = {
+  address: PropTypes.objectOf(PropTypes.string).isRequired,
+};
+
+/**
+ * @constant Address- reusable component that formats any given address object
+ */
+
+export class Address extends Component {
+  addressContainsTruthyValues = (object) =>
+    // address_type_code is always "CAD" && if "suit_no" exists but nothing else, this field isn't helpful
+    Object.keys(object)
+      .filter((key) => key !== "address_type_code" && key !== "suite_no")
+      .some((key) => object[key]);
+
+  formatRowContent = (itemArr) =>
+    itemArr
+      .filter((item) => item)
```

Comment:
What datatype is each `item` here? Are they all strings?

If we know they'll be strings, I would like to see a type-specific check here (`item !== ""`). This is more self-documenting for this fairly abstract function.

Reply:
The data will either be a string or null, never an empty string.

Reply:
Okay. So, should we strictly check against `null`? I'm just leery of implicit type coercion because it can be a bit of a footgun. When we've done this filter elsewhere, it's (mostly) been in a very specific context, where it's easy to tell what the value is/would be as it passes through the filter.